extensions/ql-vscode/src/data-extensions-editor/extension-packs.ts

```diff
@@ -0,0 +1,118 @@
+import { relative } from "path";
+import { window } from "vscode";
+import { CodeQLCliServer } from "../cli";
+import { getOnDiskWorkspaceFolders, showAndLogErrorMessage } from "../helpers";
+import { ProgressCallback } from "../progress";
+
+const maxStep = 3;
```

Comment:
Hmm what do you think about renaming the file to be `extension-pack-picker` or something like that? Then `maxStep` would make sense because it would only be relevant to picking an extension pack model file.

The alternative is ofc to rename the constant to something more specific. I don't know how we see this file going so I don't have an opinion.

Reply:
The intention of this file is to only contain the extension pack/model file picker, so I've renamed it to `extension-pack-picker.ts`.